src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/RefOnly/LooseTypeExtensions.cs

```diff
@@ -50,11 +50,11 @@ internal static bool AssemblyQualifiedNameEquals(Type t1, Type t2)
             AssemblyName name1, AssemblyName name2)
         {
             AssemblyName windowsBaseName = null;
-            if (name1.Name == WindowsBase && a2 == typeof(MarkupExtension).Assembly)
+            if (name1.Name == WindowsBase && a2 == typeof(IMarkupExtension).Assembly)
```

Comment:
Here only needs an assembly instance and there is no need to migrate `MarupExtension` to `IMarkupExtension`.

Reply:
Done. thanks for review.